pkg/controllers/managementuser/rbac/namespace_handler.go

```diff
@@ -469,6 +469,9 @@ func (m *manager) createProjectNSRole(roleName, verb, ns, projectName string) er
 	// permissions and one for write. Only the write permission should get the manage-ns verb
 	if verb == projectNSEditVerb {
 		cr = addManageNSPermission(cr, projectName)
+		if psaPermission {
```

Comment:
do we really need this check? should be enough if the verb is projectNSEditVerb (*) like for manage-ns permission?

Reply:
Since `updatepsa` brings some security concerns at the project level, I thought was better to explicitly set it, so that the user should be conscious about what is doing. Does it make sense to you?